RusimbiPatrick/Prolite
# Build A full responsive User Interface for Prolite web app

### What does this PR do?
Creates a full user Interface for Prolite web app

### Description of Task to be completed?
It has the following page:

- Homepage
- Create Ad page
-  Manage Ad page
- Sign In/ sign Out page


### How should this be manually tested?
By navigating through the pages:

1. Check if all pages exist. 
2. Check if they are responsive

### What are the relevant pivotal tracker stories?
[#166819993, #167167254, #166819858, #167160231, #166820027, #166819906, #166819930]
